packages/middleware/README.md

```diff
@@ -6,39 +6,43 @@ Supports Node.js and browsers.
 
 ## Install
 
-```
-npm install @xmpp/middleware
+```js
+npm install @xmpp/plugins
 ```
 
-## Usage
+## Hooks
 
 ```js
-const {Client} = new require('@xmpp/client')
-const middleware = require('@xmpp/middlware')
+`` // match any
+`presence` // match stanza with name of presence
+`message` // match stanza with name of presence
```

Comment:
Petite erreur dans le commentaire
`message` // match stanza with name of presence => `message` // match stanza with name of presence